containers/postgresql/reset-database.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -e
+
+db_host=$1
+db_name=$2
+db_user=$3
+
+if [[ $db_user == "postgres" ]]; then
+    echo "Do not use the built-in PostgreSQL admin account to run your application!"
+    exit 1
+fi
+
+database_query="SELECT DISTINCT schema_name
+                FROM information_schema.schemata
+                WHERE schema_owner = '"$db_user"';"
+
+schema_names="$(
+        psql                                 \
+            "$db_name"                       \
+            --host "$db_host"                \
+            --username $db_user              \
+            --tuples-only                    \
+            --command "$database_query"      \
+)"
+result=$?
+
+if [[ $result != 0 ]]; then
+    # Check if query has failed
```

Comment:
This comment would not be necessary if your variable had a better name. E.g. `query_success` instead of just `result`.

Reply:
Done.